docs/adr/0005-use-github-actions-for-vsp-platform-ci.md

```diff
@@ -1,40 +1,50 @@
-# Context
+# 5. Use GitHub Actions For VSP Platform CI
 
-https://dsva.slack.com/archives/C01CJV0L9PS/p1612651359029600?thread_ts=1612651359.029600&cid=C01CJV0L9PS
+Date: 2021-02-16
 
-TL;DR: CircleCI as implemented with VA restrictions does not allow easy management of permissions on ‘Contexts’ which contain credentials which are available to the entire ‘department-of-veterans-affairs’ Github Org. Although they are not displayed in plaintext in the CircleCI UI, they are obtainable in plaintext by running a build in CircleCI and using SSH to connect to your in-progress build, and outputting ENV variables. If the entire Org has access to the context, there is no way to stop anyone in the Org from doing so.
+Whom:
+- Demian Ginther
+- Jesse House
+- Michael Fleet
+- Dror Matalon
+## Status
+
+Accepted
+
+## Context
+
+[Slack discussion thread](https://dsva.slack.com/archives/C01CJV0L9PS/p1612651359029600?thread_ts=1612651359.029600&cid=C01CJV0L9PS).
+
+### CircleCI + VA Security
+
+CircleCI as implemented with VA restrictions does not allow easy management of permissions on ‘Contexts’ which contain credentials which are available to the entire ‘department-of-veterans-affairs’ Github Org. Although they are not displayed in plaintext in the CircleCI UI, they are obtainable in plaintext by running a build in CircleCI and using SSH to connect to your in-progress build, and outputting ENV variables. If the entire Org has access to the context, there is no way to stop anyone in the Org from doing so.
```

Comment:
I would add here that CircleCI contexts can be restricted based on github teams. However:
1) Only github organizational admins can add such restrictions - thus it must be explicitly explicitly requested
2) There isn't as yet a process for this; as such, it would be possible that even a restricted context could have an inappropriate team added to it, due to uncertainty around who 'owns' a context.

Reply:
I would add:
* Github is already ATOed
* Github is fully supported and paid for by the VA, so there should be no licensing issues.